### PR TITLE
Handle i128 arg types for arm64

### PR DESCRIPTION
**Pull request by kumarak**
Ignore `xzr` register for arm64 and unsupported registers in x86 binaries. Handle i128 parameter types in arm64.

### CODE REVIEW

File: anvill/python/anvill/binja/bnfunction.py [hunk 1]
```diff
@@ -152,6 +163,9 @@ def _extract_types(
         rdi, rsi as operands in the MLIL, we should check if they have pointer information)
         """
         results = []
+        if not (item_or_list and str(item_or_list)):
```
Comment (pgoodman) [inline anvill/python/anvill/binja/bnfunction.py:166]: What is going on here with the `str`?

File: anvill/python/anvill/binja/bnfunction.py [hunk 2]
```diff
@@ -198,7 +215,7 @@ def _fill_bytes(self, program, memory, start, end, ref_eas):
                 seg = program.bv.get_segment_at(ea)
                 br.seek(ea)
 
-                #NOTE(artem): This is a workaround for binary ninja's fake
+                # NOTE(artem): This is a workaround for binary ninja's fake
```
Comment (pgoodman) [inline anvill/python/anvill/binja/bnfunction.py:218]: Thanks :-)

**Review by pgoodman — APPROVED** [review]

File: anvill/python/anvill/binja/bnfunction.py (hunk 1 above)
Comment (kumarak) [reply anvill/python/anvill/binja/bnfunction.py:166]: `item_or_list` may not be an empty object but it's string representation may be empty. In that case there is possibility that register name is empty for unimplemented operands.

Comment (pgoodman) [reply anvill/python/anvill/binja/bnfunction.py:166]: Huh, can you add a comment for this. That is rather subtle.

**Pull request merged**